README.md

```diff
@@ -138,6 +137,45 @@ Below is a typical example of how this use case would take place:
 
 - Compute the new root for the vault’s rewards distributor, submit it, wait for the timelock (if any), accept the root, and let vault depositors claim their rewards according to the vault manager’s rewards re-distribution strategy.
 
+## Emergency
+
+### An enabled market is now considered unsafe
+
+If an enabled market is considered unsafe (e.g., risk too high), the curator/owner may want to disable this market in the following way:
+
+- Revoke the pending cap of the market with the `revokePendingCap` function (this can also be done by the guardian).
```

Comment:
maybe we should say that transactions must be batched together via the multicall?

Reply:
Maybe not "must", because it sometimes works without the multicall, so something like: "To ensure that submit cap does not revert because of a pending cap, it is recommended to batch the two previous transactions, for example using the `multicall` function of MetaMorpho."

Reply:
If done in multiple tx, the item should be "withdraw as much supply as possible with reallocate".